tests/clients/test_mysqlclient.py

```diff
@@ -217,3 +217,93 @@ def test_error_capture(self):
+
+    def test_connect_ctx_mngr(self):
+        result = None
+
+        with tracer.start_active_span('test'):
+            with self.db as conn:
+                cursor = conn.cursor()
+                result = cursor.execute("""SELECT * from users""")
+                cursor.fetchone()
+
+        assert(result >= 0)
+
+        spans = self.recorder.queued_spans()
+        self.assertEqual(2, len(spans))
+
+        db_span = spans[0]
+        test_span = spans[1]
+
+        self.assertEqual("test", test_span.data["sdk"]["name"])
+        self.assertEqual(test_span.t, db_span.t)
+        self.assertEqual(db_span.p, test_span.s)
+
+        self.assertEqual(None, db_span.ec)
+
+        self.assertEqual(db_span.n, "mysql")
+        self.assertEqual(db_span.data["mysql"]["db"], testenv['mysql_db'])
+        self.assertEqual(db_span.data["mysql"]["user"], testenv['mysql_user'])
+        self.assertEqual(db_span.data["mysql"]["stmt"], 'SELECT * from users')
+        self.assertEqual(db_span.data["mysql"]["host"], testenv['mysql_host'])
+        self.assertEqual(db_span.data["mysql"]["port"], testenv['mysql_port'])
+
+    def test_cursor_ctx_mngr(self):
+        result = None
+
+        with tracer.start_active_span('test'):
+            with self.db.cursor() as cursor:
+                result = cursor.execute("""SELECT * from users""")
+                cursor.fetchone()
+            self.db.close()
+
+        assert(result >= 0)
+
+        spans = self.recorder.queued_spans()
+        self.assertEqual(2, len(spans))
+
+        db_span = spans[0]
+        test_span = spans[1]
+
+        self.assertEqual("test", test_span.data["sdk"]["name"])
+        self.assertEqual(test_span.t, db_span.t)
+        self.assertEqual(db_span.p, test_span.s)
+
+        self.assertEqual(None, db_span.ec)
+
+        self.assertEqual(db_span.n, "mysql")
+        self.assertEqual(db_span.data["mysql"]["db"], testenv['mysql_db'])
+        self.assertEqual(db_span.data["mysql"]["user"], testenv['mysql_user'])
+        self.assertEqual(db_span.data["mysql"]["stmt"], 'SELECT * from users')
+        self.assertEqual(db_span.data["mysql"]["host"], testenv['mysql_host'])
+        self.assertEqual(db_span.data["mysql"]["port"], testenv['mysql_port'])
+
+    def test_connect_cursor_ctx_mngr(self):
+        result = None
+
+        with tracer.start_active_span('test'):
+            with self.db as conn:
+                with conn.cursor() as cursor:
+                    result = cursor.execute("""SELECT * from users""")
+                    cursor.fetchone()
+
+        assert(result >= 0)
+
+        spans = self.recorder.queued_spans()
+        self.assertEqual(2, len(spans))
+
+        db_span = spans[0]
+        test_span = spans[1]
+
+        self.assertEqual("test", test_span.data["sdk"]["name"])
+        self.assertEqual(test_span.t, db_span.t)
+        self.assertEqual(db_span.p, test_span.s)
+
+        self.assertEqual(None, db_span.ec)
```

Comment:
This should be [assertIsNone](https://docs.python.org/3/library/unittest.html?highlight=assertisnone#unittest.TestCase.assertIsNone)